src/main/scad/yavuz-demirhan/burrloop/burrloop.scad

```diff
@@ -0,0 +1,34 @@
+include <puzzlecad.scad>
+
+$burr_scale = 9;
+$plate_width = 120;
+$burr_inset = 0.05;
+$joint_inset = 0.08;
+$burr_bevel = 1;
+$unit_beveled = false;
+
+*frame();
+*pieces();
+
+module frame() {
+    render(convexity = 2)
+    burr_plate([
+        ["x{connect=mx-}xxxx|x{connect=mx-}xxxx|...xx|...x{connect=fz+}x{connect=fz+}"],
+        ["xxxxx{connect=mx+}|xxxxx{connect=mx+}|xx...|x{connect=fz+}x{connect=fz+}..."],
+            ["x{connect=mx-}xxxx|x{connect=mx-}xxxx|...xx|...x{connect=fz+}x{connect=fz+}"],
+        ["xxxxx{connect=mx+}|xxxxx{connect=mx+}|xx...|x{connect=fz+}x{connect=fz+}..."],
+            ["x{connect=mx-}xxxx|x{connect=mx-}xxxx|...xx|...x{connect=fz+}x{connect=fz+}"],
+        ["xxxxx{connect=mx+}|xxxxx{connect=mx+}|xx...|x{connect=fz+}x{connect=fz+}..."]
```

Comment:
Suggestion: use mx-z- and mx+z- connectors here (and orient the female connectors accordingly). This makes it more obvious how to assemble the frame, and it also minimizes support scarring on the joints.

If they're printed with support angle > 45 degrees, they will be self-supporting so that the scarring only affects the tip. This makes the joints fit much more cleanly & precisely - if modeled this way, then the very large $joint_inset shouldn't be necessary.

Reply:
Thanks, modified.

Actually the default joint_inset is too tight for my printer, but because I know it depends on different printer, so I also removed the setting of joint_inset.
In this kind of puzzles, it needs glues to assemble the frame, no matter how tight setting for the joints, in fact, I prefer looser joint inset in this puzzle.

Reply:
Ok, will update later